packages/augur-core/source/contracts/trading/ZeroXTrade.sol

```diff
@@ -212,7 +213,7 @@ contract ZeroXTrade is Initializable, IZeroXTrade, IERC1155 {
 
         transferFromAllowed = true;
 
-        uint256 _protocolFee = 150000 * tx.gasprice;
+        uint256 _protocolFee = exchange.protocolFeeMultiplier().mul(tx.gasprice);
```

Comment:
This will allow the 0x multisig to set the fee to whatever they want, and the fee goes to wherever the multisig wants.  What protections will be in place against a compromised/malicious 0x multisig account?  Will the UI provide some protections against this (e.g., hard code the fee in the UI so a change requires a UI update)?

Reply:
Yes the UI will. The UI will always inform the user what fees are being charged.